worker-workflow/src/main/java/com/github/cafdataprocessing/workflow/ArgumentsManager.java

```diff
@@ -53,7 +60,7 @@ public ArgumentsManager(final SettingsApi settingsApi, final String settingsServ
         Objects.requireNonNull(settingsApi);
         Objects.requireNonNull(settingsServiceUrl);
         this.settingsApi = settingsApi;
-        final OkHttpClient okHttpClient = new OkHttpClient();
+        okHttpClient = new OkHttpClient();
```

Comment:
Is's just my personal preference but please use `this.` when setting instance variables in constructors.

```suggestion
        this.okHttpClient = new OkHttpClient();
```